data_processing/steam_demand.ipynb

```diff
@@ -0,0 +1,838 @@
+      "text/plain": [
+       "<Figure size 432x288 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "steam15[(steam15['eDNA Status as String'] == 'UNRELIABLE') & (steam15.index.month <= 1)].plot()"
```

Comment:
What's this for? Does it plot just the unreliable data?

Reply:
Yes, it plots all of the data, and then it plots the data points marked as "unreliable." I wanted to see if the "unreliable" data was really that far off the mark. It seems that most of them are false positives.